composer.json

```diff
@@ -19,8 +19,16 @@
     },
     "minimum-stability": "dev",
     "prefer-stable": true,
+    "scripts": {
+        "test": "./vendor/bin/phpunit --bootstrap ./VariableAnalysis/Tests/bootstrap.php --printer=LimeDeck\\\\Testing\\\\Printer ./VariableAnalysis/Tests"
```

Comment:
Would it not be better to put these args into a `phpunit.xml.dist` file?

Reply:
Maybe! I just dislike writing xml 😛  I suppose that would simplify this line. Added #18